### PR TITLE
*: update copyright to 2017

### PR DESCRIPTION
**Pull request by holys**
@andelf  @GregoryIan  PTAL

### CODE REVIEW

**Comment by GregoryIan** [comment]
LGTM

File: syncer/config.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2016 PingCAP, Inc.
```
Comment (andelf) [inline syncer/config.go]: do not update this.
copyright `year` normally means creation year.
or  use `year-present`

Comment (andelf) [inline syncer/config.go]: like
`` Copyright (c) 2016-present, PingCAP, Inc. ``

**Comment by holys** [comment]
abandoned

**Pull request closed without merging**